### ISSUE TITLE
Export and Import features not available anymore?

### ISSUE THREAD

**Issue by ngregoire** [simioni87/auth_analyzer#42]
The features allowing to import from / export to JSON files seem to have disappear (tested on v1.1.13 from December 2022). Am I missing a new button or menu?

**Comment by simioni87** [comment]
You are right ngregoire, this feature was available through the drop down menu on the top of the burpsuite window. The "hack" to set this menu from the extension context is not working anymore. I will add this below the Settings button with the next release.

**Comment by ngregoire** [comment]
Thanks in advance!

**Comment by ngregoire** [comment]
Here's how Hackvertor creates its own entry in the menu bar https://github.com/hackvertor/hackvertor/blob/e7c9f8154976f689cf869a0a3ee3e52d43553551/src/main/java/burp/BurpExtender.java#L211-L224

Note that `getBurpFrame()` is defined in the same file https://github.com/hackvertor/hackvertor/blob/e7c9f8154976f689cf869a0a3ee3e52d43553551/src/main/java/burp/BurpExtender.java#L1187-L1194

On the other side, the new Montoya API includes a helper named `menuBar()`
https://portswigger.github.io/burp-extensions-montoya-api/javadoc/burp/api/montoya/ui/UserInterface.html#menuBar()
